idunn/geocoder/models/params.py

```diff
@@ -39,15 +37,28 @@ class QueryParams:
     zone_type: List[ZoneType] = Query([])
     poi_type: List[str] = Query([])
 
+    nlu: bool = settings["AUTOCOMPLETE_NLU_DEFAULT"]
+
     def bragi_query_dict(self):
         """
         Return a dictionary similar to the result of self.dict() but rename
```

Comment:
I am not sure this would be helpful. The existing model defines the parameters that are likely to be used by applications and are mostly back-compatible with Bragi.  
But from now, Idunn and Bragi query params may differ in the future.